package.json

```diff
@@ -10,9 +10,10 @@
       "node node_modules/react-native/local-cli/cli.js run-android --appId org.prideinlondon.festival.dev --variant devDebug",
     "test": "jest src --maxWorkers=2",
     "test:e2e": "yarn test:e2e:build && yarn test:e2e:run",
-    "test:e2e:dev": "detox test --reuse",
+    "test:e2e:dev":
+      "detox test --reuse --loglevel verbose --debug-synchronization 1000",
     "test:e2e:run":
-      "detox test -c ios.sim.debug --loglevel verbose --debug-synchronization",
+      "detox test -c ios.sim.debug --loglevel verbose --debug-synchronization 1000",
```

Comment:
1000 means that anything that takes more than 1000ms to synchronize will be logged. Great way to see network latency being an issue.